setup.cfg

```diff
@@ -87,7 +89,23 @@ console_scripts =
     bentoml = bentoml_cli.cli:cli
 
 [options.extras_require]
+grpc =
+    # Restrict maximum version due to breaking protobuf 4.21.0 changes
+    # (see https://github.com/protocolbuffers/protobuf/issues/10051)
+    protobuf>=3.5.0, <3.20
+    # Lowest version that support 3.10
+    grpcio>=1.41.0
+    grpcio-health-checking
+    grpcio-reflection
+    opentelemetry-instrumentation-grpc==0.33b0
+jaeger =
```

Comment:
Should we call this `tracing-jaeger`, so it's clear what it's for?

Reply:
I think `jaeger` is fine

Reply:
Yeah, but I think `tracing-jaeger` is more descriptive. @ssheng maybe for a third opinion?

Reply:
Did you chat about this one?

Reply:
not yet.